extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -47,160 +74,366 @@ Because of standardization of markup, support for multiple providers, and improv
+The Entitlement response returned by the authorization endpoint must conform to the predefined format:
+<table>
+  <tr>
+    <th>Property</th>
+    <th>Values</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>granted</code></td>
+    <td>&lt;boolean&gt;</td>
+    <td>Stating whether or not the Reader has access to the document or not.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>grantReason</code></td>
+    <td>&lt;string&gt;</td>
+    <td>The reason for giving the access to the document, recognized reasons are: <ul><li><code>"SUBSCRIBER"</code> meaning the user is fully subscribed.</li><li><code>"METERING"</code> meaning user is on metering.</li></ul></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>data</code></td>
+    <td>&lt;object&gt;</td>
+    <td>Free-form data which can be used for template rendering, e.g. messaging related to metering or article count. See <a href="#customising-content">Customising Content</a> for more details.</td>
+  </tr>
+</table>
+
+Example response for a Reader who is a subscriber and is logged into their account:
 
-## Service configuration
+```js
+{
+  "granted": true,
+  "grantReason": "SUBSCRIBER",
+  "data" : {
+    "isLoggedIn": true
+  }
+}
+```
 
-The `amp-subscriptions` extension must be configured using JSON configuration:
+Example response for an anonymous Reader who has read 4 out of 5 free articles:
 
-```html
-<script type="application/json" id="amp-subscriptions">
-  {
-    "services": [
-      {
-        // Service 1 (local service)
-      },
-      {
-        // Service 2 (a vendor service)
-      }
-    ],
-    "score": {
-      "supportsViewer": 10,
-      "isReadyToPay": 9
-    },
-    "fallbackEntitlement": {
-      "source": "fallback",
-      "granted": true,
-      "grantReason": "SUBSCRIBER/METERING",
-      "data": {...}
-    }
+```js
+{
+  "granted": true,
+  "grantReason": "METERING",
+  "data" : {
+    "isLoggedIn": false,
+    "articlesRead": 4,
+    "articlesLeft": 1,
+    "articleLimit": 5
   }
-</script>
+}
 ```
 
-The `services` property contains an array of service configurations. There must be one "local" service and zero or more vendor services.
+Example response for an anonymous Reader who does not have access because they have read 5 out of 5 free articles:
 
-If you'd like to test the document's behavior in the context of a particular viewer, you can add `#viewerUrl=` fragment parameter. For instance, `#viewerUrl=https://www.google.com` would emulate a document's behavior inside a Google viewer.
+```js
+{
+  "granted": false,
+  "data" : {
+    "isLoggedIn": false,
+    "articlesRead": 5,
+    "articlesLeft": 0,
+    "articleLimit": 5
+  }
+}
+```
 
-## Selecting a service
+Notice, while it's not explicitly visible, all vendor services also implement authorization endpoints of their own and conform to the same response format.
```

Comment:
Where would I have noticed this? Is it pointed out somewhere?

Reply:
It's implicit from the description in the "Flow" section where we say:

> The AMP Runtime calls the Authorization endpoint of all configured services.

Reply:
Maybe reword to be more clear:
`All vendor services must implement authorization endpoint of their own and conform to the amp-subscriptions [response format](link to response format here please).`